.github/workflows/test-and-release-beta.yml

```diff
@@ -30,6 +30,14 @@ jobs:
           githubToken: ${{ secrets.GITHUB_TOKEN  }}
```

Comment:
Could you add a name also to this one, like "Test All Checks"

Reply:
Done!